src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -232,18 +207,18 @@ static void uct_rdamcm_cm_ep_destroy_reserved_qpn(uct_rdmacm_cm_ep_t *cep)
 
     ucs_debug("cm ep destroy reserved qpn on rdmacm_id %p", cep->id);
 
-    status = uct_rdmacm_cm_get_reserved_qpn_context(uct_rdmacm_cm_ep_get_cm(cep),
-                                                    cep->id->verbs, &ctx);
+    status = uct_rdmacm_cm_get_qp_context(uct_rdmacm_cm_ep_get_cm(cep),
+                                          cep->id->verbs, &ctx);
     if (status != UCS_OK) {
-        ucs_error("get reserved qpn context failed");
-        return;
+        ucs_error("destroy reserved qpn failed");
     }
+    ucs_assert(status == UCS_OK);
```

Comment:
something is redundant here

Reply:
@evgeny-leksikov It was added to avoid compile error since ucs_assert can be defined as blank. Have removed the ucs_error code and changed ucs_assert to ucs_assert_always.